src/group_theory/subgroup.lean

```diff
@@ -1078,6 +1078,33 @@ lemma le_normalizer_of_normal [hK : (H.comap K.subtype).normal] (HK : H ≤ K) :
   λ yH, by simpa [mem_comap, mul_assoc] using
              hK.conj_mem ⟨x * y * x⁻¹, HK yH⟩ yH ⟨x⁻¹, K.inv_mem hx⟩⟩
 
+variables {N : Type*} [group N]
+
+/-- The preimage of the normalizer is contained in the normalizer of the preimage -/
```

Comment:
Here and below, doc strings which are complete sentences should end with periods.
```suggestion
/-- The preimage of the normalizer is contained in the normalizer of the preimage. -/
```